code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -991,3 +998,26 @@
 	H.ContractDisease(P)
 	parrot_interest = null
 	H.visible_message("<span class='danger'>[src] dive bombs into [H]'s chest and vanishes!</span>", "<span class='userdanger'>[src] dive bombs into your chest, vanishing! This can't be good!</span>")
+
+
+/mob/living/simple_animal/parrot/clock_hawk
+	name = "clock hawk"
+	desc = "Cbyl jnaan penpxre! Fdhnnnjx!"
+	icon_state = "clock_hawk_fly"
+	icon_living = "clock_hawk_fly"
+	icon_sit = "clock_hawk_sit"
+	speak = list("Penpxre!", "Ratvar vf n qhzo anzr naljnl!")

```

Comment:
Tvir zr n penpxre, puhpxyrshpx!

Reply:
I hope xhuis is doing alright after his horrible keyboard strokes.